contrib/cirrus/setup_environment.sh

```diff
@@ -58,6 +58,9 @@ cd "${GOSRC}/"
 # Reload to incorporate any changes from above
 source "$SCRIPT_BASE/lib.sh"
 
+echo ">>>>> WORKAROUND/HACK (05/30/19): metacopy=on breaks --privileged + VFS tests"
```

Comment:
Looks like you need `if [ -e /etc/containers/storage.conf ]`

Also, sorry, what does metacopy and --privileged have to do with this?

Reply:
Good idea.  It was only --privleged tests using VFS that failed, but maybe that's just how those tests are setup.  I'll remove that part of the comment.